invenio_app_rdm/config.py

```diff
@@ -174,6 +174,9 @@ def _(x):
 THEME_SITENAME = _("InvenioRDM")
 """Site name."""
 
+THEME_SHOW_FRONTPAGE_INTRO_SECTION = True
```

Comment:
why do we need this config variable?

Reply:
Discussed with @lnielsen to have this config variable to easily disable the intro-section